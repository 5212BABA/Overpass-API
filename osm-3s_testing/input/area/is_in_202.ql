is_in(51.2599,7.1501);out geom;
