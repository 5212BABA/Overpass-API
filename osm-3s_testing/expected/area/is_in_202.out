<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base=""/>

  <way id="202">
    <bounds minlat="51.2500001" minlon="7.1500001" maxlat="51.2600001" maxlon="7.1510001"/>
    <nd ref="20201" lat="51.2500001" lon="7.1500001"/>
    <nd ref="20202" lat="51.2500001" lon="7.1510001"/>
    <nd ref="20203" lat="51.2600001" lon="7.1510001"/>
    <nd ref="20204" lat="51.2600001" lon="7.1500001"/>
    <nd ref="20201" lat="51.2500001" lon="7.1500001"/>
  </way>

</osm>
